app/src/org/commcare/logic/PendingCalloutInterface.java

```diff
@@ -16,5 +16,9 @@
 public interface PendingCalloutInterface {
     FormIndex getPendingCalloutFormIndex();
 
+    boolean isPendingCalloutBulk();
```

Comment:
Removed that concept, sorry, I forgot that I had changed how I was preventing that response processing.